pkg/watch/registrar.go

```diff
@@ -75,15 +76,15 @@ func (r *recordKeeper) NewRegistrar(parentName string, events chan<- event.Gener
 }
 
 // RemoveRegistrar removes a registrar and all its watches.
-func (r *recordKeeper) RemoveRegistrar(parentName string) error {
+func (r *recordKeeper) RemoveRegistrar(ctx context.Context, parentName string) error {
```

Comment:
It looks like this context will no longer be used after the other contexts are removed?

Reply:
Done.